packaging/python/build_wheels.bash

```diff
@@ -52,15 +52,16 @@ pip_numpy_install() {
       36) numpy_ver="numpy==1.12.1" ;;
       37) numpy_ver="numpy==1.14.6" ;;
       38) numpy_ver="numpy==1.17.5" ;;
-      39) numpy_ver="numpy==1.19.3" ;;
-      310) numpy_ver="numpy==1.21.3" ;;
-      311) numpy_ver="numpy==1.23.5" ;;
-      312) numpy_ver="numpy==1.26.0" ;;
+      39) numpy_ver="numpy>=2" ;;
+      310) numpy_ver="numpy>=2" ;;
+      311) numpy_ver="numpy>=2" ;;
+      312) numpy_ver="numpy>=2" ;;
+      313) numpy_ver="numpy>=2" ;;
```

Comment:
This is because numpy 2 is backwards compatible, but numpy 1 is not forwards compatible, so building a wheel with numpy 1.x and testing it with 2.x will fail, but the other way around works.